src/main/resources/static/css/cartDetail.css

```diff
@@ -0,0 +1,99 @@
+.mdl-dialog {
+    top: 45%;
+    width: 33%;
+}
+
+.mdl-dialog__actions {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between
+}
+
+.cart-item-display-override {
+    display: block;
+    padding: 0;
+}
+
+.cart-product-item {
+    margin: 1em 0;
+    width: 60%;
+}
+
+.cart-product-item-title {
+    line-height: 36px;
+    padding-left: 6px;
+    font-size: 24px;
+    font-weight: 500;
+    letter-spacing: .04em;
+}
+
+.cart-product-item-content {
+    line-height: 36px;
+    font-size: 20px;
+    font-weight: 500;
+    letter-spacing: .04em;
+    padding: 2px;
```

Comment:
please place comment -> /*shared styling*/ which contains classes already used by other CSS-files. 
please place comment -> /*file specific styling*/ which contains classes only used by this specific file. 
See other CSS-files for reference. I'm no frontender either, but we can at least try to keep it seperable :)